client/src/components/WeekListItem/WeekListItem.js

```diff
@@ -29,14 +48,24 @@ const TextWeek = styled.a`
 `;
 
 const WeekListItem = () => {
+  const [date, setDate] = useState(new Date());
+
+  const getWeekDay = (date) => {
+    const week = ["So", "Mo", "Di", "Mi", "Do", "Fr", "Sa"];
+    const weekday = week[date.getDay()];
+    return weekday;
+  };
```

Comment:
if you need the short version of the weekday, in german, of the actual date, there is an automated way
```suggestion
  const getWeekDay = (date) => {
    return (new Intl.DateTimeFormat('de-DE', {weekday: "short"}).format(date))
  };
```

Reply:
oh wow, didn't know that. thanks! :) :)